internal/admission/validator.go

```diff
@@ -304,6 +306,59 @@ func (validator KongHTTPValidator) ValidateGateway(
 	return true, "", nil
 }
 
+func (validator KongHTTPValidator) ValidateHTTPRoute(
+	ctx context.Context, httproute gatewayv1alpha2.HTTPRoute,
+) (bool, string, error) {
+	if len(httproute.Spec.ParentRefs) == 0 {
+		// we don't try to validate an HTTPRoute if it doesn't have parentRefs
+		// configured, because there's no way we can be sure the route is managed
+		// by this controller implementation.
+		return true, "", nil
+	}
```

Comment:
redundant? the loop below will yield the same result.

Reply:
sure, changed in 2078fb5a133a95b37654a2bb8891e4a7b966f27f